### PR TITLE
cursor plugin: add showTimeYPosition option

### PR DESCRIPTION
**Pull request by thijstriemstra**

### CODE REVIEW

**Comment by thijstriemstra** [comment]
@katspaugh not sure about the option name and option values, any thoughts or is it clear enough?

File: src/plugin/cursor.js [hunk 1]
```diff
@@ -13,6 +13,9 @@
  * @property {boolean} showTime=false Show the time on the cursor.
  * @property {object} customShowTimeStyle An object with custom styles which are
  * applied to the cursor time element.
+ * @property {string} showTimeYPosition='fixed' Use 'relative' to make the time on
```
Comment (katspaugh) [inline src/plugin/cursor.js]: `showTimeYPosition` sounded like a boolean value until I read the description.
Maybe `followCursorY {boolean}`?

**Comment by thijstriemstra** [comment]
@katspaugh can you adjust the coveralls coverage percentage? So it does not show up as failed when a tiny percentage, e.g. 0.3%, coverage decrease is detected? I think 1 or 1.5% is a better threshold.

**Comment by katspaugh** [comment]
@thijstriemstra I've set it to 1.5% 👌

**Pull request merged**